src/pages/RequestCallPage.js

```diff
@@ -96,6 +92,28 @@ class RequestCallPage extends Component {
         return secondaryLogin ? Str.removeSMSDomain(secondaryLogin.partnerUserID) : null;
     }
 
+    /**
+     * Gets the first and last name from the user's personal details.
+     * If the login is the same as the displayName, then they don't exist,
+     * so we return empty strings instead.
+     * @param {String} login
+     * @param {String} displayName
+     *
+     * @returns {Object}
+     */
+    getFirstAndLastName({login, displayName}) {
+        if (login === displayName) {
+            return {firstName: '', lastName: ''};
+        }
+
+        const firstSpaceIndex = displayName.indexOf(' ');
+        const lastSpaceIndex = displayName.lastIndexOf(' ');
+        return {
+            firstName: displayName.substring(0, firstSpaceIndex),
+            lastName: firstSpaceIndex !== lastSpaceIndex ? displayName.substring(lastSpaceIndex) : '',
```

Comment:
are you sure about this logic?
Example: `Andre Fonseca`
firstSpaceIndex = 5
lastSpaceIndex = 5
If I'm getting the logic right this would be `firstName: "Andre"` `lastName:""`

Reply:
Ah you're right, I intended it to be `firstSpaceIndex !== -1`, thanks for catching this!